<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDlgSInFReg</class>
 <widget class="CItemDlg" name="CDlgSInFReg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>118</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Body InnerMouth Region Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>CheckRegion:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDrawReg">
       <property name="text">
        <string>Set</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnResetReg">
       <property name="text">
        <string>ResetReg</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_13">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>194</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>Gray</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinGray">
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>OpenSize</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spinOpenSize">
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="minimum">
        <double>0.500000000000000</double>
       </property>
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.500000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_21">
       <property name="text">
        <string>Area</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinArea">
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupPos">
       <property name="font">
        <font>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QRadioButton" name="radioLeft">
          <property name="whatsThis">
           <string/>
          </property>
          <property name="text">
           <string>Left</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioRight">
          <property name="text">
           <string>Right</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CItemDlg</class>
   <extends>QWidget</extends>
   <header>citemdlg.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>btnDrawReg</tabstop>
  <tabstop>spinGray</tabstop>
  <tabstop>spinArea</tabstop>
  <tabstop>radioLeft</tabstop>
  <tabstop>radioRight</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <slots>
  <slot>accept()</slot>
  <slot>reject()</slot>
 </slots>
</ui>
